packages/xo-web/src/common/xo/index.js

```diff
@@ -1595,28 +1595,22 @@ export const deleteTemplates = templates =>
     )
 
     const nProtectedTemplates = protectedTemplates.length
-    const showError = () =>
-      error(
-        _('failedToDeleteTemplatesTitle', { nTemplates: nErrors }),
-        _('failedToDeleteTemplatesMessage', { nTemplates: nErrors })
-      )
-
-    const protectedDefaultTemplates = new Set()
+    let forceBlockedOperation = false
 
     if (nProtectedTemplates !== 0) {
       await confirm({
         title: _('deleteProtectedTemplatesTitle', { nProtectedTemplates }),
         body: _('deleteProtectedTemplatesMessage', { nProtectedTemplates }),
       })
+      forceBlockedOperation = true
```

Comment:
So you are passing `forceBlockedOperation` even if that not necessary?

Reply:
Yes, but with the correct value.

Reply:
E.g. You delete 5 templates. 1 is default and protected, 4 is only default.
In reallity, you don't need to send `forceBlockedOperation: true` for the 4 default templates

Reply:
Yes, but it doesn't hurt to pass it even when you know the operation isn't blocked. The point was only to simplify the code.